ingest/expression_files/mtx.py

```diff
@@ -23,7 +23,7 @@
 from bson.objectid import ObjectId
 
```

Comment:
let's exclude changes to this file from this commit, so we can focus our full attention on dense